src/main/java/uk/gov/di/ipv/cri/passport/service/PassportService.java

```diff
@@ -43,18 +46,19 @@ public PassportService()
         this.httpClient = HttpClientSetUp.generateHttpClient(configurationService);
     }
 
-    public DcsSignedEncryptedResponse dcsPassportCheck(JWSObject payload) throws IOException {
+    public Optional<DcsSignedEncryptedResponse> dcsPassportCheck(JWSObject payload)
+            throws IOException {
         HttpPost request = new HttpPost(configurationService.getDCSPostUrl());
-        request.addHeader("content-type", "application/jose");
+        request.addHeader(CONTENT_TYPE, APPLICATION_JOSE);
         request.setEntity(new StringEntity(payload.serialize()));
 
         HttpResponse response = httpClient.execute(request);
 
         if (response == null) {
-            throw new NullPointerException("Response from DCS is null");
+            return Optional.empty();
```

Comment:
It think I was thinking more along the lines of:
```
Optional<HttpResponse> response = Optional.ofNullable(httpClient.execute(request));

if (response.isEmpty()) {
    throw new SomeException();
}

return new DcsSignedEncryptedResponse(response.toString());
```

Also, will httpClient.execute() ever actually return null?